src/libnyoci/assert-macros.h

```diff
@@ -79,6 +80,12 @@
 				PSTR(__FILE__ ":%d: "fmt"\n"), \
 				__LINE__, \
 				__VA_ARGS__)
+ #elif ESP_PLATFORM
```

Comment:
I like all of the changes in this file, including the `errno` stuff. May be worth spinning out.